inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -285,6 +285,17 @@ def is_submission(obj, eng):
     return get_method(obj.data) == 'submitter'
 
 
+def is_feature_flag_enabled(key):
```

Comment:
is this still used? also what you wrote in the commit message doesn't match the content of the PR.

Reply:
this is not the final version